doc/ModulesDevelopment.md

```diff
@@ -16,11 +16,11 @@
       * [Test run](#test-run)
       * [Modification of a Task](#modification-of-a-task)
       * [Check](#check)
-        * [Configuration](#configuration)
-        * [Implementation](#implementation)
+         * [Configuration](#configuration)
+         * [Implementation](#implementation)
       * [Committing code](#committing-code)
 
-<!-- Added by: bvonhall, at:  -->
+<!-- Added by: barth, at: Lun 16 mar 2020 13:18:24 CET -->
```

Comment:
It looks like you have some clever way to update the table of contents automatically. How do you do that?

Reply:
I download this software : https://github.com/ekalinin/github-markdown-toc

And then run it on the file.